examples/app/polymer/polymer.jsf.js

```diff
@@ -0,0 +1,39 @@
+var app = angular.module('makeithappen');
+app.run(['RenderService', 'PathResolver', '$rootScope', function(RenderService, PathResolver, $rootScope) {
+
+    function PolymerControl() {
+
+        var instance;
+        var path;
+
+        // set-up event binding
+        document.addEventListener('WebComponentsReady', function() {
```

Comment:
i don't like this part at all ... but if the control doesn't support databinding there is no other way :-(